src/components/AirmassTelescopeStates.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div class="airmassTelescopeStatesCombo">
```

Comment:
Also thought we were excluding the telescope_states plots because they use LCO specific ES telemetry...